pkg_client/src/components/APIHandler.tsx

```diff
@@ -1,50 +1,26 @@
-import React, { useContext, useEffect, useState } from "react";
-import axios from "axios";
+import React, { useContext } from "react";
 import { UserContext } from "../contexts/UserContext";
 import Container from "react-bootstrap/Container";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
```

Comment:
Is order of the imported libraries correct?

Reply:
Imports are not reorganized based on [ESLint sort-imports rule](https://eslint.org/docs/latest/rules/sort-imports).